Kitodo/src/main/java/org/kitodo/production/services/command/OverwriteDataScript.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.production.services.command;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Objects;
+
+import org.kitodo.api.Metadata;
+import org.kitodo.api.MetadataEntry;
+import org.kitodo.api.MetadataGroup;
+import org.kitodo.api.dataformat.IncludedStructuralElement;
+import org.kitodo.api.dataformat.Workpiece;
+import org.kitodo.data.database.beans.Process;
+import org.kitodo.production.helper.metadata.legacytypeimplementations.LegacyMetsModsDigitalDocumentHelper;
+
+public class OverwriteDataScript extends EditDataScript {
+
+    @Override
+    public void executeScript(LegacyMetsModsDigitalDocumentHelper metadataFile, Process process,
+            MetadataScript metadataScript) {
+        Workpiece workpiece = metadataFile.getWorkpiece();
+        List<IncludedStructuralElement> allIncludedStructuralElements = workpiece
+                .getAllIncludedStructuralElements();
+
+        IncludedStructuralElement child = allIncludedStructuralElements.get(0);
```

Comment:
Same as above. I think you should check if `allIncludedStructuralElements` contains an element before "getting" it.